Formula/libssh.rb

```diff
@@ -22,10 +23,12 @@ class Libssh < Formula
 
   def install
     mkdir "build" do
-      system "cmake", "..", "-DWITH_STATIC_LIB=ON",
-                            "-DWITH_SYMBOL_VERSIONING=OFF",
-                            *std_cmake_args
+      system "cmake", "..", *std_cmake_args, "-DWITH_SYMBOL_VERSIONING=OFF"
       system "make", "install"
+      system "make", "clean"
+      system "cmake", "..", *std_cmake_args, "-DBUILD_SHARED_LIBS=OFF"
+      system "make"
+      lib.install "src/libssh.a"
```

Comment:
Is there no way to build this with only one `cmake` call?

Reply:
Right, let me try again.

Reply:
OK I updated it, thanks!

Reply:
Thanks. Is the `lib.install` line still needed? Shouldn't that be installed by `make install`?

Reply:
Yeah, strangely it doesn't get installed automatically, unless you build with `BUILD_SHARED_LIBS=OFF`.

Reply:
That's a bit troubling. Is it possible upstream actually don't intend for them to be built simultaneously for some reason?

Reply:
It's actually how it works for most libraries, that you can install either shared or static libs, depending on the value of `BUILD_SHARED_LIBS`. They have done this on purpose: https://github.com/libssh/libssh-mirror/commit/729c92606c88b60e7c255d393756d1d3b7e30698

Reply:
Yes, but I was assuming you used to be able to install both... they can't have done this on purpose for no reason, right?